datasource/opennms-kafka/src/main/java/org/opennms/alec/datasource/opennms/OpennmsDatasource.java

```diff
@@ -585,7 +597,7 @@ private <K,V> ReadOnlyKeyValueStore<K, V> waitUntilStoreIsQueryable(String store
 
     @Override
     public void forwardSituation(Situation situation) {
-        if (situation.getAlarms().size() < 1) {
+        if (situation.getAlarms().isEmpty()) {
```

Comment:
nit: the interface _Situation_ doesn't guarantee that getAlarms() is not null. Having said that: The only implementation _ImmutableSituation_ does.

Reply:
Sonar was complaining about using size() < 1 instead of isEmpty()

Reply:
yeah I think we are good here...